rasa/train.py

```diff
@@ -639,12 +647,14 @@ async def _train_nlu_with_validated_data(
         )
 
         if model_to_finetune:
-            model_to_finetune = _nlu_model_for_finetuning(
-                model_to_finetune, config, finetuning_epoch_fraction
+            model_to_finetune = await _nlu_model_for_finetuning(
+                model_to_finetune,
+                file_importer,
+                finetuning_epoch_fraction,
+                also_train_core=train_path is not None,
```

Comment:
let's add a comment why this works. I doubt that our future selves will remember how this plays together. Or even better sth like `called_from_combined_training = train_path is None`